psi4/driver/driver.py

```diff
@@ -553,12 +559,15 @@ def energy(name, **kwargs):
             restartfile = (restartfile, )
         # Rename the files to be read to be consistent with psi4's file system
         for item in restartfile:
+            is_numpy_file = "npy" in item or os.path.isfile(item + ".npy")
```

Comment:
If we want to catch files that end with "npy", perhaps `item.endswith(".npy")` would work better. We should also check that that file exists, so:
`is_numpy_file = (os.path.isfile(item) and item.endswith(".npy")) or os.path.isfile(item + ".npy")`